packages/image-block/src/ImageBlock.spec.ct.tsx

```diff
@@ -84,31 +85,21 @@ describe('Image Block', () => {
     it('should render the title if it is provided', () => {
         const [ImageBlockWithStubs] = withAppBridgeBlockStubs(ImageBlock, {
             blockSettings: {
-                name: 'Test Name',
+                name: JSON.stringify([{ type: 'imageTitle', children: [{ text: 'Test Name' }] }]),
             },
         });
         mount(<ImageBlockWithStubs />);
-        cy.get(ImageBlockSelector).should('contain.text', 'Test Name');
-    });
-
-    it('should render the file name if no title is provided', () => {
```

Comment:
Add a test where no title is setted and the asset gets uploaded and afterwards the asset title is the title, so we have that case also

Reply:
Newly we set the title after uploading and this unfortunately cannot be mockt at the moment with the appBridge asset dummy